hedera-mirror-rest/topicmessage.js

```diff
@@ -400,6 +409,39 @@ const getMessages = async (pgSqlQuery, pgSqlParams, preQueryHint) => {
   return rows.map((row) => new TopicMessage(row));
 };
 
+const getQueryWithTopicMessageLookupTimestamps = async (topicId, seqNum) => {
+  // Get the timestamp range from the topic message lookup table
+  const sequenceNumber = BigInt(seqNum);
+  const topicMessageLookupQuery = `select lower(timestamp_range) as timestamp_start, upper(timestamp_range) as timestamp_end
+                    from ${TopicMessageLookup.tableName}
+                    where topic_id = $1 and sequence_number_range @> $2::int8
```

Comment:
nit, for consistency, we use bigint everywhere
```suggestion
                    where topic_id = $1 and sequence_number_range @> $2::bigint
```